### PR TITLE
update axios and frontend-platform

### PR DESCRIPTION
**Pull request by muselesscreator**
update axios and frontend-platform which depends on it to resolve a security issue.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
   "dependencies": {
     "@edx/brand": "npm:@edx/brand-edx.org@^1.3.2",
     "@edx/frontend-component-footer": "10.1.1",
-    "@edx/frontend-platform": "^1.8.0",
+    "@edx/frontend-platform": "1.8.1",
```
Comment (nsprenkle) [inline package.json:31]: Is there a reason to pin this at `1.8.1` instead of `^1.8.1`?

Comment (nsprenkle) [reply package.json:31]: LGTM pending an answer on this line.

**Review by nsprenkle — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (mattcarter) [reply package.json:31]: Let's keep that convo moving, but elsewhere. I don't think pinning/not pinning is a block to removing the vulnerable version.

**Review by mattcarter — APPROVED** [review]
LGTM.

**Pull request merged**